appserver/extras/payara-micro/payara-micro-core/src/main/java/fish/payara/micro/impl/PayaraMicroImpl.java

```diff
@@ -2204,6 +2204,11 @@ private void setArgumentsFromSystemProperties() {
         if (alternateHZConfigFileStr != null && !alternateHZConfigFileStr.isEmpty()) {
             alternateHZConfigFile = new File(alternateHZConfigFileStr);
         }
+        
+        String userLogPropertiesFileStr = getProperty("payaramicro.logPropertiesFile");
+        if (userLogPropertiesFileStr  != null && !userLogPropertiesFileStr.isEmpty()) {
```

Comment:
How abot calling !userLogPropertiesFileStr.trim().isEmpty() to make sure it is not a blank string?